R/ohsome_parse.R

```diff
@@ -1,39 +1,46 @@
 #' Parse content from an ohsome API response
 #'
-#' Extract and parse the content from an ohsome API response
+#' Extracts and parses the content from an ohsome API response
 #'
-#' \code{ohsome_parse()} parses an \code{ohsome_response} object into
-#' an object of the specified class. By default, this is an \code{sf} object if
-#' the ohsome API response contains GeoJSON data or a \code{data.frame} if it
-#' does not. \code{ohsome_sf()}  and \code{ohsome_df()} are wrapper functions.
+#' `ohsome_parse()` parses an `ohsome_response` object into an object of the 
+#' specified class. By default, this is an `sf` object if the ohsome API 
```

Comment:
rephrase the sentence." By default, this is an \code{sf} object if
#' the ohsome API response contains GeoJSON data or a \code{data.frame} if it
#' does not"